homeassistant/components/lutron_caseta/strings.json

```diff
@@ -1,5 +1,21 @@
 {
   "config": {
-    "title": "Lutron Caséta"
+    "title": "Lutron Caséta",
+    "step": {
+      "user": {
+        "title": "Connect to the device",
+        "data": {
+          "host": "Host"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "Failed to connect, please try again",
+      "invalid_auth": "Invalid authentication",
+      "unknown": "Unexpected error"
```

Comment:
Just keep the strings that we use. Update the flow first to connect then check if we should remove anything more here.